### PR TITLE
[FrameworkBundle] Show server:run logs by default

### PR DESCRIPTION
**Pull request by nicolas-grekas**
| Q | A |
| --- | --- |
| Branch? | master |
| Bug fix? | no |
| New feature? | yes |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | - |
| License | MIT |
| Doc PR | - |

I propose to change the default for the `server:run` command and show `php -S` logs by default.
I really miss them otherwise. The `-vvv` mode is not suited here, because it adds a useless `ERR` red prefix.
I do this through a tty when available, so that the output remains colored.

Ping @javiereguiluz @weaverryan since this is mostly a DX issue.

### CODE REVIEW

File: src/Symfony/Bundle/FrameworkBundle/Command/ServerRunCommand.php [hunk 1]
```diff
@@ -114,14 +117,22 @@ protected function execute(InputInterface $input, OutputInterface $output)
...
-        if (OutputInterface::VERBOSITY_VERBOSE > $output->getVerbosity()) {
-            $process->disableOutput();
+        try {
+            if (OutputInterface::VERBOSITY_NORMAL > $output->getVerbosity()) {
+                $process->disableOutput();
+            } else {
+                $process->setTty(true);
+            }
+            $process->run();
+        } catch (RuntimeException $e) {
```
Comment (stof) [inline src/Symfony/Bundle/FrameworkBundle/Command/ServerRunCommand.php]: couldn't runtime exceptions happen for other reasons than unsupported tty mode during ``$process->run();``, causing weird cases of running the process twice ?

Comment (nicolas-grekas) [inline src/Symfony/Bundle/FrameworkBundle/Command/ServerRunCommand.php]: let's be safe, fixed

File: src/Symfony/Bundle/FrameworkBundle/Command/ServerRunCommand.php [hunk 2]
```diff
@@ -114,13 +117,22 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $builder->setTimeout(null);
         $process = $builder->getProcess();
 
-        if (OutputInterface::VERBOSITY_VERBOSE > $output->getVerbosity()) {
-            $process->disableOutput();
+        try {
+            if (OutputInterface::VERBOSITY_NORMAL > $output->getVerbosity()) {
+                $process->disableOutput();
```
Comment (stof) [inline src/Symfony/Bundle/FrameworkBundle/Command/ServerRunCommand.php]: do we need to have this inside the try/catch ?

Comment (nicolas-grekas) [inline src/Symfony/Bundle/FrameworkBundle/Command/ServerRunCommand.php]: no but we need to have this `if/else` logic so code is easier this way

Comment (stof) [inline src/Symfony/Bundle/FrameworkBundle/Command/ServerRunCommand.php]: the try/cache can be inside the ``else`` (and defining ``$callback = null;`` before the ``if``)

Comment (nicolas-grekas) [inline src/Symfony/Bundle/FrameworkBundle/Command/ServerRunCommand.php]: done

**Comment by fabpot** [comment]
Thank you @nicolas-grekas.

**Pull request merged**